files/jquery.query-builder/update.json

```diff
@@ -0,0 +1,9 @@
+{
+  "packageManager": "bower",
+  "name": "jQuery-QueryBuilder",
+  "repo": "mistic100/jQuery-QueryBuilder",
+  "files": {
+    "basePath": "dist/",
+    "include": ["**/*"]
```

Comment:
Its a soft requirement. Unless you can prove you need the excluded files you should use the one I provided.